code/game/objects/items/weapons/grenades/concussion.dm

```diff
@@ -74,14 +64,12 @@
 						H.sdisabilities |= SDISABILITY_DEAF
 				else if(H.ear_damage >= 5)
 					to_chat(H, "<span class='danger'>Your ears start to ring!</span>")
+			/* Until someone can come up with a better thing to have happen to borgs like a stun or something this is getting commented out
```

Comment:
How about all modules in the "hot bar" getting put back into storage?

Reply:
I have no idea how to do that in a way that's not jank